.travis.yml

```diff
@@ -21,6 +21,7 @@ install:
 
 # running tests
 script:
+  - cd /tmp
   - pytest -v --maxfail=2 --cov=jupyterhub jupyterhub/tests
```

Comment:
A bit more info: 

Did the location of CSS files change (or stay the same) with the bower-lite changes? I commented out part of the `test_static_files` that was causing the test failure and all passed.

```
 #r = yield async_requests.get(ujoin(base_url, 'static', 'css', 'style.min.css'))
 #r.raise_for_status()
 #assert r.headers['content-type'] == 'text/css'
```

[Passing tests on Travis](https://travis-ci.org/willingc/jupyterhub/jobs/261604492)